### PR TITLE
Increase memory allocation to paymentservice to prevent OOMKills

### PR DESCRIPTION
**Pull request by irvingpop**
I was hitting the same issues as #596 - and based on my observations of the process under light load the RSS is in the range of 78-84 MB, but even at cold startup it's above the 70MB limit.

I propose increasing to 100Mi so there's a bit of safety margin here. 

Signed-off-by: Irving Popovetsky <irving@honeycomb.io>

### CODE REVIEW

**Comment by TylerHelmuth** [comment]
@puckpuck I know you've been doing extensive stress testing of the demo chart, do you already have a PR ready that handles https://github.com/open-telemetry/opentelemetry-helm-charts/issues/596 as well as any other fixes you've discovered with your testing?

**Comment by puckpuck** [comment]
I do have a PR ready with improvements to memory limits for a few services, including Payment.  it will be part of the 1.3 release.  I can break it out as a separate PR, but I think we will cut 1.3 tomorrow.

**Comment by TylerHelmuth** [comment]
@puckpuck unless there is something performance-related to 1.3's image I'd vote for breaking it out.  Our 1.2 app version of the chart could use the performance tweaks now.

**Comment by TylerHelmuth** [comment]
If paymentservice is the main culprit, we can move forward with this PR as the solution.

**Comment by puckpuck** [comment]
It should be 120Mi ... that's what my PR is setting it to.

File: charts/opentelemetry-demo/values.yaml [hunk 1]
```diff
@@ -386,7 +386,7 @@ components:
         value: "8080"
     resources:
       limits:
-        memory: 70Mi
+        memory: 100Mi
```
Comment (TylerHelmuth) [inline charts/opentelemetry-demo/values.yaml:389]:
```suggestion
        memory: 120Mi
```

**Review by TylerHelmuth — COMMENTED** [review]
@irvingpop please bump the chart's patch version and then run `make generate-examples CHARTS=opentelemetry-demo`.  To generate the examples you'll need the chart's dependencies added to your local help repos.

**Comment by irvingpop** [comment]
thanks for the tip @TylerHelmuth I hopefully did all that correctly.

**Comment by irvingpop** [comment]
whoops now I see what you mean about bumping the chart version.

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**